helm/servicex/templates/app/configmap.yaml

```diff
@@ -98,7 +98,7 @@ data:
     TRANSFORMER_SIDECAR_IMAGE = "{{ .Values.transformer.sidecarImage }}:{{ .Values.transformer.sidecarTag }}"
     TRANSFORMER_SIDECAR_PULL_POLICY = '{{ .Values.transformer.sidecarPullPolicy }}'
 
-    TRANSFORMER_SCIENCE_IMAGE = "{{ .Values.transformer.defaultScienceContainerImage }}:{{ .Values.transformer.defaultScienceContainerTag }}"
+    TRANSFORMER_SCIENCE_IMAGE = "{{ .Values.codeGen.defaultScienceContainerImage }}:{{ .Values.codeGen.defaultScienceContainerTag }}"
```

Comment:
You will need to update the [helm publish](https://github.com/ssl-hep/ServiceX/blob/b9a6622cc32291f04e0145f49fded534e7476eb4/.github/workflows/ci_helm_publish.yaml#L55) GitHub action workflow

Reply:
I have changed the helm ci for the merge
The changes made are here
Before -
.transformer.defaultScienceContainerTag = strenv(DOCKER_TAG) | .transformer.defaultScienceContainerTag style="double" |

After -
.codeGen.defaultScienceContainerTag = strenv(DOCKER_TAG) | .codeGen.defaultScienceContainerTag style="double" |